app/src/pages/inside/logsPage/logItemInfo/InfoTabs/InfoTabs.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Parser from 'html-react-parser';
 import ArrowDownIcon from 'common/img/arrow-down-inline.svg';
-import styles from './logItemInfoTabs.scss';
+import styles from './InfoTabs.scss';
```

Comment:
Oh, just one more point we've missed: `InfoTabs` -> `infoTabs` - we use camelCase for file names.

Reply:
Yes, right. Fixed.